packages/larcore/package.py

```diff
@@ -90,6 +93,7 @@ def fetch_remote_versions(self, concurrency=None):
     depends_on("larcorealg")
     depends_on("art-root-io")
     depends_on("cetmodules", type="build")
+    depends_on("cetbuildtools", type="build" , when="@:09.11")
```

Comment:
Looks to me like this should be 

```python
depends_on("cetbuildtools", type="build" , when="@:09.05.00")
```

based on the [commit history](https://github.com/LArSoft/larcore/commit/bac1d631aa01fc4df11370566221ea4f8167967f).

Reply:
But it was the failure to build 09.10.01 that prompted the change...

Reply:
@marcmengel, that's weird.  Nothing in version `09.10.01` depends on `cetbuildtools`.  Maybe it's an underlying dependency somewhere?